go.mod

```diff
@@ -15,41 +15,45 @@ require (
 	github.com/jackc/pgx/v4 v4.18.2
 	github.com/mwitkow/grpc-proxy v0.0.0-20230212185441-f345521cb9c9
 	github.com/olebedev/emitter v0.0.0-20190110104742-e8d1457e6aee
-	github.com/open-policy-agent/opa v0.44.0
+	github.com/open-policy-agent/opa v0.68.0
 	github.com/pborman/uuid v1.2.0
 	github.com/qri-io/jsonpointer v0.1.1
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rs/cors v1.8.3
-	github.com/sirupsen/logrus v1.9.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/smart-core-os/sc-api/go v1.0.0-beta.45
 	github.com/smart-core-os/sc-golang v0.0.0-20240930110121-7fcd731e2877
 	github.com/timshannon/bolthold v0.0.0-20210913165410-232392fc8a6a
 	github.com/vanti-dev/gobacnet v0.0.0-20231102122752-32b0b38bcc53
-	go.etcd.io/bbolt v1.3.6
-	go.uber.org/multierr v1.8.0
-	go.uber.org/zap v1.19.1
+	go.etcd.io/bbolt v1.3.10
+	go.uber.org/multierr v1.9.0
+	go.uber.org/zap v1.21.0
 	golang.org/x/crypto v0.26.0
 	golang.org/x/exp v0.0.0-20240823005443-9b4947da3948
-	golang.org/x/oauth2 v0.13.0
+	golang.org/x/oauth2 v0.21.0
 	golang.org/x/sync v0.8.0
 	golang.org/x/term v0.23.0
-	golang.org/x/time v0.3.0
+	golang.org/x/time v0.6.0
 	golang.org/x/tools v0.24.0
-	google.golang.org/grpc v1.60.1
-	google.golang.org/protobuf v1.33.0
+	google.golang.org/grpc v1.66.0
```

Comment:
Wasn't sure about this.

I **unnofficially counted** more `panic: Drain not implemented` errors during ctrl + c commands to terminate sc-bos backend. This is after I previously updated `google.golang.org/grpc` to `v1.61+`. I never empirically assessed this error as the Drain function wasn't implemented in `v.1.60.1` either. 🤷‍♂️